client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -159,7 +159,7 @@ const DataFilesCopyModal = React.memo(() => {
               Destination
               <DataFilesSystemSelector
                 operation="copy"
-                systemId={params.system}
+                systemId={(systems[0] || params).system}
```

Comment:
Should this instead reference the "default" storage system rather than the first storage system in the list? It might make it more fault tolerant to configuration changes on the backend.

Reply:
I think this needs to be a new discussion/ticket, we don't have a "default" system readily defined in the frontend (afaik) and this is the "google drive styling" ticket and not the "everything wonky with modal listings" ticket.